ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -228,15 +210,9 @@ impl<P: Parameters> AffineCurve for GroupAffine<P> {
         })
     }
 
-    #[inline]
-    fn mul<S: Into<<Self::ScalarField as PrimeField>::BigInt>>(&self, by: S) -> GroupProjective<P> {
-        let bits = BitIteratorBE::new(by.into());
-        self.mul_bits(bits)
-    }
-
     #[inline]
     fn mul_by_cofactor_to_projective(&self) -> Self::Projective {
-        self.scale_by_cofactor()
+        self.mul_bits(BitIteratorBE::new(P::COFACTOR))
     }
```

Comment:
I guess now that `mul_bits` is available in the trait, we can move this implementation to the trait impl. This would further deduplicate code across the TE and SW affine impls.
```rust
pub trait AffineCurve {
	// rest
	
	#[inline]
    fn mul_by_cofactor_to_projective(&self) -> Self::Projective {
        self.mul_bits(BitIteratorBE::new(P::COFACTOR))
    }
}
```

(And the same for `mul_by_cofactor_inverse`.).